db/migrate/20150629192719_add_password_reset_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddPasswordResetToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :password_reset_token, :string
```

Comment:
You could add an index to this table in order to aid lookup in the `find_by_password_reset_token` used in your password reset controller. 

`add_index :users, :password_reset_token`